content/authors/authors.yml

```
@@ -39,3 +39,30 @@
       url: https://www.instagram.com/andriawan_irwan/
     - name: linkedin
       url: https://www.linkedin.com/in/andriawan
+      
+
+
+- name: Darvin Sinaga
+  slug: darvin-sinaga
+  bio: |
+    Darvin adalah seorang Frontend Developer Tampan di prospace.io.
+    Saat ini ia sangat tertarik dan menekuni React dan  ekosistemnya.
+    Aktif di beberapa komunitas online maupun offline.
+
+    http://darvinsinaga.com
+  avatar: ./avatars/darvin.jpg
+  featured: true
```

```suggestion
  featured: false
```
Please don't feature yourself before contributing to any article.
`featured` here means that your profile will be put in the home page.

---

oke sure